### PR TITLE
Install docker-compose (via pip) and optionally add users to the docker group

### PR DESCRIPTION
**Pull request by gildegoma**
For sake of merge simplicity, I first packed all this stuff in a single PR, but I am of course open to create dedicated pull requests if you prefer (or don't accept all these changes).

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -127,3 +129,12 @@
     name: docker.io
     state: started
   when: docker_pkg_name == 'docker.io'
+
+  # ATTENTION: this task can potentially create new users!
```
Comment (gildegoma) [inline tasks/main.yml]: we could provide a "safer" behaviour with a (maybe non-idempotent)  ̀command` task running some  ̀usermod` call, but I personnally prefer to rely on Ansible core modules, with this little drawback.

File: tasks/main.yml [hunk 2]
```diff
@@ -127,3 +129,12 @@
...
+- name: Add users to the docker group
+  user:
+    name:   "{{ item }}"
+    groups: docker
+    append: yes
+  with_items: docker_group_members
+  when: docker_group_members is defined
```
Comment (gildegoma) [inline tasks/main.yml]: the  ̀wheǹ condition is not strictly necessary, as I put an empty list in `defaults/main.yml`.

**Comment by angstwad** [comment]
Thanks for this.  Please give me some time to review these changes before merging.

**Comment by gildegoma** [comment]
@angstwad thanks for the acknowledgement, no hurry on my end :)

**Comment by angstwad** [comment]
This is great. Thanks.

**Pull request merged**